.github/workflows/build.yml

```diff
@@ -10,6 +10,7 @@ env:
   CACHED_DEPENDENCY_PATHS: |
     ${{ github.workspace }}/node_modules
     ${{ github.workspace }}/packages/**/node_modules
+    ~/.cache/ms-playwright/
```

Comment:
An alternative to this was to 

> Set `PLAYWRIGHT_BROWSERS_PATH=0` as the environment variable before running `npm install`. This will download the browser binaries in the `node_modules` directory and cache them with the package code.

(from https://playwright.dev/docs/ci/#caching-browsers)